filament/backend/src/opengl/OpenGLDriver.h

```diff
@@ -475,8 +475,8 @@ class OpenGLDriver final : public backend::DriverBase {
         } pack;
 
         struct {
-            vec4gli scissor = 0;
-            vec4gli viewport = 0;
+            vec4gli scissor = vec4gli(0);
```

Comment:
maybe try the `scissor = {}` or `scissor{}` instead. Hopefully one of those {} syntax will work.

Reply:
just `{ 0 };` works ... ` = { 0 };` doesn't.  I'll check it in.

Reply:
@pixelflinger done